workflow/applications/applications.py

```diff
@@ -61,7 +61,6 @@ def __init__(self, conf: Configuration) -> None:
         self.do_genesis = _base.get('DO_GENESIS', True)
         self.do_genesis_fsu = _base.get('DO_GENESIS_FSU', False)
         self.do_metp = _base.get('DO_METP', False)
-        self.do_npoess = _base.get('DO_NPOESS', False)
```

Comment:
Same, why is this being removed?

Reply:
See https://github.com/NOAA-EMC/global-workflow/pull/2103#discussion_r1415841460